.github/workflows/release-please.yaml

```diff
@@ -11,40 +11,3 @@ jobs:
       - uses: google-github-actions/release-please-action@v4
         with:
           release-type: go
```

Comment:
I think we need to add a line in the with statement: manifest-file: .release-please-manifest.json